GitUI/LeftPanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -245,7 +245,7 @@ private void contextMenu_Opening(object sender, CancelEventArgs e)
 
             if (hasSingleSelection && selectedLocalBranch?.Visible == true)
             {
-                contextMenu.AddUserScripts(runScriptToolStripMenuItem, ExecuteCommand, UICommands);
+                contextMenu.AddUserScripts(runScriptToolStripMenuItem, DispatchCommand, (script) => script.AddToRevisionGridContextMenu, UICommands);
```

Comment:
```suggestion
                contextMenu.AddUserScripts(runScriptToolStripMenuItem, ExecuteCommand, script => script.AddToRevisionGridContextMenu, UICommands);
```
needs bubbling up in `ExecuteCommand`.

Reply:
No it doesn't. `ExecuteCommand` is called (indirectly) from `ProcessCmdKey`, which is bubbled on the OS level. If `ExecuteCommand` bubbled, `ProcessCmdKey` would double-bubble. That's why I'm adding `DistpatchCommand` instead.

(But I can agree on the brackets thing)

I also prefer to use `DistpatchCommand` instead of `ExecuteCommand` even in cases where bubbling is guranteed to not happen, to have uniformity, and avoid future C&P issues.